mlperf_logging/benchmark_meta.py

```diff
@@ -131,6 +131,14 @@
         'stable_diffusion',
         'llama2_70b_lora',
         'gnn'
+    ],
+    '5.0': [
+        'bert',
+        'dlrm_dcnv2',   
+        'retinanet',        
+        'stable_diffusion',
+        'llama2_70b_lora',
+        'gnn'
```

Comment:
Rename gnn -> rgat

Reply:
Done

Reply:
I updated it for `v5.0`. Some places still reference `gnn` for backwards compatibility. For example, to run the `v4.0` or `v4.1` submission checker